test/fit_tests/deploy/os_ova_install.py

```diff
@@ -4,7 +4,11 @@
 Author(s):
 George Paulos
 
-This script installs blank OVA template.
+This script installs blank OVA templates into selected stack.
+if numvms is greater than 1, MAC address installation on OVA is done by stack number convention
```

Comment:
Sorry, I don't understand this comment - do you mean if numvms is greater than 1, you will create MAC addresses based on the stack number?  stacknumber + vm number. and if equal to 1, then uses the default mac specified in the stack config file?
thanks!